.github/container/test-pax.sh

```diff
@@ -0,0 +1,264 @@
+#!/bin/bash
+
+# # Parse command-line arguments
+
+print_var() {
+    echo "$1: ${!1}"
+}
+
+usage() {
+    echo "Test T5X throughput on a fake-data Wikipedia benchmark."
+    echo ""
+    echo "Usage: $0 [OPTIONS]"
+    echo ""
+    echo "  OPTIONS                    DESCRIPTION"
+    echo "  -b, --batch-per-gpu        Batch size per GPU, defaults to 4."
+    echo "  --dtype                    Batch size, defaults to bfloat16."
+    echo "  -s, --steps                Number of steps to run, defaults to 500."
+    echo "  --multiprocess             Enable the multiprocess GPU mode."
+    echo "  -o, --output NAME          Name for the output folder, a temporary folder will be created if none specified."
+    echo "  --data-parallel            Data parallelism to use. Defaults to 1."
+    echo "  --tensor-parallel          Tensor parallelism to use. Defaults to 1."
+    echo "  --pipeline-parallel        Pipeline parallelism to use. Defaults to 1 for no pipelining." 
+    echo "  -h, --help                 Print usage."
+    exit $1
+}
+
+args=$(getopt -o b:s:o:h --long batch-per-gpu:,dtype:,steps:,help,multiprocess,output:,data-parallel:,tensor-parallel:,pipeline-parallel: -- "$@")
+if [[ $? -ne 0 ]]; then
+    exit $1
+fi
+
+# Default arguments
+MULTIPROCESS=0
+OUTPUT=$(mktemp -d)
+
+export BATCH_PER_GPU=4
```

Comment:
It is not necessary to `export` the variables for use in the heredoc (`EOF ... EOF`). We can directly use variable substitutions (`${VARIABLE}`).

Reply:
Done. Note that paxml expects `VOCAB_PATH` to be set as an environment variable, so we still export `VOCAB_PATH` in the script.

Reply:
This script will run on a GitHub Actions runner, which will submit a SLURM job to a remote cluster. Hence, the variable will not be available even if exported here.

Reply:
We will need to add an `#SBATCH --export=...` directive around line 61-67 to pass in this variable.